overlord/devicestate/devicestate_install_mode_test.go

```diff
@@ -0,0 +1,94 @@
+// -*- Mode: Go; indent-tabs-mode: t -*-
+
+/*
+ * Copyright (C) 2019 Canonical Ltd
+ *
+ * This program is free software: you can redistribute it and/or modify
+ * it under the terms of the GNU General Public License version 3 as
+ * published by the Free Software Foundation.
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU General Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public License
+ * along with this program.  If not, see <http://www.gnu.org/licenses/>.
+ *
+ */
+
+package devicestate_test
+
+import (
+	. "gopkg.in/check.v1"
+
+	"github.com/snapcore/snapd/overlord/devicestate"
+	"github.com/snapcore/snapd/overlord/state"
+	"github.com/snapcore/snapd/release"
+)
+
+type deviceMgrInstallModeSuite struct {
+	deviceMgrBaseSuite
+}
+
+var _ = Suite(&deviceMgrInstallModeSuite{})
+
+func (s *deviceMgrInstallModeSuite) findInstallSystem() *state.Change {
+	for _, chg := range s.state.Changes() {
+		if chg.Kind() == "install-system" {
+			return chg
+		}
+	}
+	return nil
+}
+
+func (s *deviceMgrInstallModeSuite) TestInstallModeCreatesChangeHappy(c *C) {
+	s.state.Lock()
+	defer s.state.Unlock()
+
+	release.OnClassic = false
+	s.state.Set("seeded", true)
```

Comment:
This could also move to a new "SetUpTest()" as it seems to be common accross most of the tests.

Reply:
Done.